app/main.py

```
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    arguments_list = command.split()
+    if arguments_list[0] == "cp" and len(arguments_list) == 3:
```

This violates the checklist item: "Make sure you check that the input string contains 3 elements separated by space, and has `cp` on a first place." 

Why: the condition `if arguments_list[0] == "cp" and len(arguments_list) == 3:` (line 3) accesses `arguments_list[0]` before confirming the list has the required number of tokens. If `command` is empty or has fewer tokens this will raise `IndexError` instead of safely returning.

Suggested fix: validate the token count before indexing. For example, after the single `arguments_list = command.split()` call, check `if len(arguments_list) != 3 or arguments_list[0] != "cp": return` and then unpack the two file names. This preserves the single `.split()` requirement and ensures robust validation as required by the checklist.